src/inverseRobot.js

```diff
@@ -24,7 +24,27 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const arr = [];
+
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    for (let k = 0; k < i; k++) {
+      if ((values[i] === values[k]) === true) {
```

Comment:
```suggestion
     if (values[i] === values[k]) {
```

 
Оскільки if повертає true або false в залежності від умови, то прирівнювати додатково до true немає необхідності